### PR TITLE
Slicing Login Page

### PR DESCRIPTION
**Pull request by intannmn**
[
Screenshot 2023-11-21 092207
](url)
Slicing Login Page :
- Add Harsa Logo
- Feat Login Page

### CODE REVIEW

File: src/pages/login/index.jsx
Comment (NizanHulq) [inline src/pages/login/index.jsx:1]: Kayaknya coba handling form nya pakai react hook form deh, biar kedepannya kalau form itu handling input dan validationnya pakai RHF

**Review by NizanHulq — COMMENTED** [review]
Handling form input nya bisa pakai RHF, bisa diliat dokumentasi shadcn form with RHF di link ini : 
https://ui.shadcn.com/docs/components/form

File: src/pages/login/index.jsx
Comment (intannmn) [reply src/pages/login/index.jsx:1]: udah saya ubah pakai rhf, tolong dicek lagi mas nizan

Comment (intannmn) [reply src/pages/login/index.jsx:1]: https://github.com/HarsaEdu/harsa-frontend/assets/126959335/32395f8d-b577-4df9-8d63-f061a2b1cef0

Udah aku update pakai rhf mas nizan, tolong dicek kembali ya.

**Review by KokoYuardiA — COMMENTED** [review]
langsung lanjut ke intregrasi aja mbak, meet discord malam minggu jam 8

**Comment by intannmn** [comment]
Screenshot 2023-11-25 224130

Revisi :
- Update schema validation with zod
- Update form with shadcn

**Pull request merged**